pallets/protos/src/lib.rs

```diff
@@ -392,6 +395,7 @@ pub mod pallet {
 			tags: Vec<Vec<u8>>,
 			linked_asset: Option<LinkedAsset>,
 			license: UsageLicense<T::AccountId>,
+			cluster: Option<Cluster<T::AccountId>>,
```

Comment:
Open continuation issue to implement proto side

Reply:
#239